src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -340,6 +340,8 @@ internal override TypeSymbol IteratorElementType
 
         public override DllImportData GetDllImportData() => null;
 
+        public override bool AreLocalsZeroed => throw ExceptionUtilities.Unreachable;
```

Comment:
I think all of these can be replaced with a single override that returns `true` in `SourceMethodSymbol`

Reply:
Just talked with @AlekseyTs and he thinks we should be explicit in each callsite. I think that's a fine decision as well and we can go with that.

However, I would be very surprised if this didn't throw whenever we try to emit local functions, so I think we'll have to provide a real implementation, even if it's accompanied by a prototype comment saying that the full spec will be implemented in another PR.

Reply:
To answer my own question, by the time we end up at creating a MethodBody, LocalFunctions and similar are all turned into SynthesizedInstance or similar symbols, so throwing is actually the right thing to do here, I think.